### ISSUE TITLE
local-up 18.1 crashes on --type hetzner

### ISSUE THREAD

**Issue by batthebee** [edgefarm/edgefarm#81]
```bash
local-up cluster create --type hetzner                                                                                                                                                 INT ✘ 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x23c6933]

goroutine 1 [running]:
github.com/edgefarm/edgefarm/pkg/config/v1alpha1.Validate(0x0)
	/home/runner/work/edgefarm/edgefarm/pkg/config/v1alpha1/validate.go:50 +0x33
github.com/edgefarm/edgefarm/cmd/local-up/cmd.NewCreateCommand.func1(0xc000005500, {0xc0004b4540, 0x0, 0x2})
	/home/runner/work/edgefarm/edgefarm/cmd/local-up/cmd/clusterCreate.go:100 +0x7a5
github.com/spf13/cobra.(*Command).execute(0xc000005500, {0xc0004b4520, 0x2, 0x2})
	/home/runner/work/edgefarm/edgefarm/vendor/github.com/spf13/cobra/command.go:940 +0xcce
github.com/spf13/cobra.(*Command).ExecuteC(0x38794c0)
	/home/runner/work/edgefarm/edgefarm/vendor/github.com/spf13/cobra/command.go:1068 +0x9ed
github.com/spf13/cobra.(*Command).Execute(0x38794c0)
	/home/runner/work/edgefarm/edgefarm/vendor/github.com/spf13/cobra/command.go:992 +0x32
github.com/edgefarm/edgefarm/cmd/local-up/cmd.Execute()
	/home/runner/work/edgefarm/edgefarm/cmd/local-up/cmd/root.go:36 +0x1e
main.main()
	/home/runner/work/edgefarm/edgefarm/cmd/local-up/main.go:34 +0x215
```

**Comment by batthebee** [comment]
Maybe same issue: 

local-up cluster create                                                                                                                                                                                                             ✔ 
I0312 17:34:49.215210  287008 create.go:169] Start to prepare config for kind
apiVersion: kind.x-k8s.io/v1alpha4
kind: Cluster
name: edgefarm
networking:
  disableDefaultCNI: true
nodes:
  - role: control-plane
    kubeadmConfigPatches:
    - |
      kind: ClusterConfiguration
      apiServer:
        extraArgs:
          kubelet-preferred-address-types: InternalIP,Hostname,ExternalDNS,ExternalIP
      controllerManager:
        extraArgs:
          cluster-signing-duration: 87600h0m0s
      kubernetesVersion: v1.22.17
    - |
      kind: KubeletConfiguration
      cgroupDriver: systemd
    image: ghcr.io/edgefarm/edgefarm/kind-node:v1.22.7-systemd
    extraPortMappings:
    - containerPort: 6443
      hostPort: 6443
  - role: worker
    image: ghcr.io/edgefarm/edgefarm/kind-node:v1.22.7-systemd
    extraPortMappings:
    - containerPort: 4222
      hostPort: 4222
      listenAddress: "0.0.0.0"
    - containerPort: 80
      hostPort: 80
      listenAddress: "0.0.0.0"
    - containerPort: 443
      hostPort: 443
      listenAddress: "0.0.0.0"
    labels:
      ingress-ready: "true"
    kubeadmConfigPatches:
    - |
      kind: ClusterConfiguration
      kubernetesVersion: 
    - |
      kind: KubeletConfiguration
      cgroupDriver: systemd
  - role: worker
    image: ghcr.io/edgefarm/edgefarm/kind-node:v1.22.7-systemd
    labels:
      openyurt.io/is-edge-worker: "true"
    kubeadmConfigPatches:
    - |
      kind: ClusterConfiguration
      kubernetesVersion: 
    - |
      kind: KubeletConfiguration
      cgroupDriver: systemd
  - role: worker
    image: ghcr.io/edgefarm/edgefarm/kind-node:v1.22.7-systemd
    labels:
      openyurt.io/is-edge-worker: "true"
    kubeadmConfigPatches:
    - |
      kind: ClusterConfiguration
      kubernetesVersion: 
    - |
      kind: KubeletConfiguration
      cgroupDriver: systemd
I0312 17:34:49.215377  287008 create.go:175] Start to create cluster with kind
Creating cluster "edgefarm" ...
 • Ensuring node image (ghcr.io/edgefarm/edgefarm/kind-node:v1.22.7-systemd) 🖼  ...
WARNING: Overriding docker network due to KIND_EXPERIMENTAL_DOCKER_NETWORK
WARNING: Here be dragons! This is not supported currently.
 ✓ Ensuring node image (ghcr.io/edgefarm/edgefarm/kind-node:v1.22.7-systemd) 🖼
 • Preparing nodes 📦 📦 📦 📦   ...
 ✓ Preparing nodes 📦 📦 📦 📦 
 • Writing configuration 📜  ...
 ✓ Writing configuration 📜
 • Starting control-plane 🕹️  ...
 ✓ Starting control-plane 🕹️
 • Installing StorageClass 💾  ...
 ✓ Installing StorageClass 💾
 • Joining worker nodes 🚜  ...
 ✓ Joining worker nodes 🚜
I0312 17:35:54.160299  287008 create.go:180] Cluster created
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x23d869e]

goroutine 1 [running]:
github.com/edgefarm/edgefarm/pkg/clusters/local.(*Initializer).Run(0xc0002341e0)
	/home/runner/work/edgefarm/edgefarm/pkg/clusters/local/create.go:181 +0x31e
github.com/edgefarm/edgefarm/pkg/clusters/local.CreateCluster()
	/home/runner/work/edgefarm/edgefarm/pkg/clusters/local/create.go:56 +0x10c
github.com/edgefarm/edgefarm/cmd/local-up/cmd.NewCreateCommand.func1(0xc000005b00, {0x38d4620, 0x0, 0x0})
	/home/runner/work/edgefarm/edgefarm/cmd/local-up/cmd/clusterCreate.go:93 +0x6fd
github.com/spf13/cobra.(*Command).execute(0xc000005b00, {0x38d4620, 0x0, 0x0})
	/home/runner/work/edgefarm/edgefarm/vendor/github.com/spf13/cobra/command.go:940 +0xcce
github.com/spf13/cobra.(*Command).ExecuteC(0x38794c0)
	/home/runner/work/edgefarm/edgefarm/vendor/github.com/spf13/cobra/command.go:1068 +0x9ed
github.com/spf13/cobra.(*Command).Execute(0x38794c0)
	/home/runner/work/edgefarm/edgefarm/vendor/github.com/spf13/cobra/command.go:992 +0x32
github.com/edgefarm/edgefarm/cmd/local-up/cmd.Execute()
	/home/runner/work/edgefarm/edgefarm/cmd/local-up/cmd/root.go:36 +0x1e
main.main()
	/home/runner/work/edgefarm/edgefarm/cmd/local-up/main.go:34 +0x215

**Comment by siredmar** [comment]
Thanks for reporting these. 
Both cases are indeed separate issues. I'll take a look and report back.

**Comment by siredmar** [comment]
fixed in v1.9.x
